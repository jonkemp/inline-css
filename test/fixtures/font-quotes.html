<html>
<head>
    <style>
        p {
            font-family: 'Open Sans', Arial, sans-serif;
        }
    </style>
</head>
<body>
    <p style="">hi</p>
    <p style=''>there</p></body></html>
