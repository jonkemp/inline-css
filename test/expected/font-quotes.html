<html>
<head>
    
</head>
<body>
    <p style="font-family: 'Open Sans', Arial, sans-serif;">hi</p>
    <p style="font-family: 'Open Sans', Arial, sans-serif;">there</p>
</body>
</html>
